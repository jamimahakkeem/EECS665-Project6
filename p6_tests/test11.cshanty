int main() {
	int a;
	a = 1;
	if(a < 1) {
		return 1;
	}
	else {
		return 0;
	}

	bool b;
	b = true;
	return 10;
}
