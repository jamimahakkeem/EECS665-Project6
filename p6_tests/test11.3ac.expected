[BEGIN GLOBALS]
[END GLOBALS]
[BEGIN main LOCALS]
a (local var of 8 bytes)
b (local var of 8 bytes)
tmp0 (tmp var of 8 bytes)
[END main LOCALS]
main:       enter main
            [a] := 1
            [tmp0] := [a] LT64 1
            IFZ [tmp0] GOTO lbl_1
            [a] := 2
            goto lbl_2
lbl_1:      nop
            [a] := 3
lbl_2:      nop
            [b] := 1
            setret 10
            goto lbl_0
lbl_0:      leave main
